client/src/curriculum/module.scss

```
@@ -21,7 +29,7 @@
         }
       }
 
-      p {
+      p.module-topic {
```

Nit: Do we need to be this explicit?

```suggestion
      .module-topic {
```

---

No, but we can. That's a discussion for having CSS guidelines in the future.